ajax/libs/simple-icons/package.json

```diff
@@ -1,34 +1,26 @@
 {
   "name": "simple-icons",
-  "filename": "css3.svg",
+  "filename": "simpleicons.svg",
   "version": "3.0.1",
-  "description": "PNG and SVG icons for popular brands",
+  "description": "SVG icons for popular brands https://simpleicons.org",
   "homepage": "https://simpleicons.org",
   "keywords": [
     "icons",
-    "SVG",
-    "PNG",
-    "image"
+    "svg"
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/danleech/simple-icons.git"
+    "url": "git+ssh://git@github.com/simple-icons/simple-icons.git"
```

Comment:
Let's use `https://github.com/simple-icons/simple-icons.git` or cdnjs website will fail to bring users to the expected web page.